torch/csrc/jit/type.h

```diff
@@ -370,6 +371,32 @@ struct TORCH_API CompleteTensorType : public TensorType {
   std::vector<int64_t> strides_;
 };
 
+// This type is a token used to represent effectful computation in the IR.
+// See the AnnotateEffects pass for how it is used.
+struct WorldType;
```

Comment:
Do a search for `GeneratorType` in the codebase. There are a few switch statements that are missing clauses to handle the WorldType (such as module import/export).